pkg/operator/ceph/cluster/controller.go

```diff
@@ -116,7 +118,10 @@ func newReconciler(mgr manager.Manager, ctx *clusterd.Context, clusterController
 		panic(err)
 	}
 
-	clusterController.recorder = k8sutil.NewEventReporter(mgr.GetEventRecorderFor("ClusterController"))
+	// add "rook-" prefix to the controller name to make sure it is clear to all reading the events
+	// that they are coming from Rook. The controller name already has context that it is for Ceph
+	// and from the cluster controller.
+	clusterController.recorder = k8sutil.NewEventReporter(mgr.GetEventRecorderFor("rook-" + controllerName))
```

Comment:
What's an example of how the events look? Adding the "rook-" prefix seems redundant since it's only on the rook CRs.

Reply:
Events can come from anywhere, so it is important to fully denote where they are coming from.

Reply:
Just curious, what an example of the events looks like?

Reply:
Most I see come from `kubelet` or from `default-scheduler`. Upstream trends seem to be to give this a name that is `lower-case-with-dashes`.

Reply:
As a follow-up thought, It could be a reasonable idea to have a higher-level recorder that isn't at the controller level that identifies all events as `rook-ceph` since I don't think we need to give that granularity to users or for debugging.

Reply:
Yeah the extra info about which controller probably isn't relevant to the user, but seems like nice debug info for anyone looking at a bug report.